src/MOL_discretization.jl

```diff
@@ -61,8 +63,8 @@ function SciMLBase.symbolic_discretize(pdesys::PDESystem, discretization::Method
     ####
     for pde in pdeeqs
         # Read the dependent variables on both sides of the equation
-        depvars_lhs = get_depvars(pde.lhs, depvar_ops)
-        depvars_rhs = get_depvars(pde.rhs, depvar_ops)
+        depvars_lhs = get_depvars(pde.lhs, v.depvar_ops)
+        depvars_rhs = get_depvars(pde.rhs, v.depvar_ops)
```

Comment:
why not flatten here?

Reply:
Can you expand? This code is inherited from diffeqops